src/content/en/2021/webassembly.md

```diff
@@ -247,7 +250,7 @@ Compression aside, we could also look for optimization opportunities by analyzin
   )
 }}
 
-Unsurprisingly, most—~74%—of the total binary size comes from the compiled code itself, followed by ~19% for embedded static data. Function types, import/export descriptors and such comprise a negligible part of the total size. However, one section type stands out—it's custom sections, which account for ~6.5% of total size in the mobile dataset.
+Unsurprisingly, ~74% of the total binary size comes from the compiled code itself, followed by ~19% for embedded static data. Function types, import/export descriptors and such comprise a negligible part of the total size. However, one section type stands out - it's custom sections, which account for ~6.5% of total size in the mobile dataset.
```

Comment:
I'd argue "most" is important here ("unsurprusingly, most" vs "unsurprisingly, ~74%" - only the fact that it's most is unsurprising, not the precise percentage).

Reply:
I think it just sounds odd right now, maybe there’s a grammar tweak needed for it to flow better with the word most. Any ideas?

Reply:
I'm probably the wrong person to ask because to me the original sentence sounds fine 😅 Wil moving % to parentheses like "most (~74%)" help?

Reply:
What about brackets?:

```suggestion
Unsurprisingly, most (~74%) of the total binary size comes from the compiled code itself, followed by ~19% for embedded static data. Function types, import/export descriptors and such comprise a negligible part of the total size. However, one section type stands out - it's custom sections, which account for ~6.5% of total size in the mobile dataset.
```

Reply:
That works!

Reply:
Or even move that further down the sentence:

```suggestion
Unsurprisingly, most of the total binary size (~74%) comes from the compiled code itself, followed by ~19% for embedded static data. Function types, import/export descriptors and such comprise a negligible part of the total size. However, one section type stands out - it's custom sections, which account for ~6.5% of total size in the mobile dataset.
```

Reply:
I like that too

Reply:
First variant seems to be exactly what I proposed, and the 2nd one looks good too. Unfortunately, looks like I can't accept either now because suggestions are marked as outdated after emdash change :(

Reply:
Updated to second in https://github.com/HTTPArchive/almanac.httparchive.org/pull/2605/commits/5129015399710c63060966d803fd930ece2d19e4